.gitignore

```diff
@@ -0,0 +1,3 @@
+/prombench
+/scaler
+/fake-webserver
```

Comment:
lets try without a .gitignore file as everyone has a different workflows.
use .git/info/excludes instead.

btw you committed some binary files.

Reply:
I deleted fake-webserver binary file.
It has been pushed some time back.

Reply:
I added .gitignore because if we make changes to the code and want to build it locally using `make`.
promu will always build the 3 binaries.
This will remain same for every workflow :)

Reply:
> I deleted fake-webserver binary file.
It had been pushed some time back.
 
yes sorry , my bad.

> I added .gitignore because if we make changes to the code and want to build it locally using make.
promu will always build the 3 binaries.
This will remain same for every workflow.
prometheus repo also has .gitignore for binaries built during make

still I prefer if we don't add a .gitignore if this can be avoided by simply adding this to  .git/info/excludes.